docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-python.md

```diff
@@ -0,0 +1,84 @@
+---
+title: How to run a Checkpoint in python
+---
+
+import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx';
+
+This guide will help you run a Checkpoint in python.
+This is useful if your pipeline environment or orchestration engine does not have shell access.
+
+:::note Prerequisites
+  This how-to guide assumes you have already:
+
+  - Created at least one [Checkpoint](./how-to-create-a-new-checkpoint).
+
+Steps
+-----
+
+1. First, generate the python with the command:
+
+```bash
+
+    great_expectations --v3-api checkpoint script my_checkpoint
+```
+
+2. Next, you will see a message about where the python script was created like:
+
+```bash
+
+    A python script was created that runs the checkpoint named: `my_checkpoint`
+      - The script is located in `great_expectations/uncommitted/run_my_checkpoint.py`
+      - The script can be run with `python great_expectations/uncommitted/run_my_checkpoint.py`
+```
```

Comment:
```suggestion
```bash
A python script was created that runs the checkpoint named: `my_checkpoint`
  - The script is located in `great_expectations/uncommitted/run_my_checkpoint.py`
  - The script can be run with `python great_expectations/uncommitted/run_my_checkpoint.py`
```
```